lib/cookie.d.ts

```diff
@@ -389,6 +393,22 @@ export class Cookie {
    * Does a deep clone of this cookie, exactly implemented as Cookie.fromJSON(cookie.toJSON()).
    */
   clone (): Cookie;
+
+  /**
+   * Status: IN PROGRESS. Works for a few things, but is by no means comprehensive.
+   *
+   * validates cookie attributes for semantic correctness. Useful for "lint" checking any Set-Cookie headers you
+   * generate. For now, it returns a boolean, but eventually could return a reason string -- you can future-proof with
+   * this construct:
+   * <pre><code>
+   * if (cookie.validate() === true) {
+   *   // it's tasty
+   * } else {
+   *   // yuck!
+   * }
+   * </code></pre>
```

Comment:
I can't recall if it's used anywhere else in the file, but would you mind using markdown-style triple backticks over HTML? I'm not sure either are currently used anywhere though.